src/docfx/resolve/Resolve.cs

```diff
@@ -80,9 +86,17 @@ private static (DocfxException error, Document file, string fragment, string que
                 pathToDocset = Path.Combine(Path.GetDirectoryName(relativeTo.FilePath), path);
             }
 
+            // resolve from redirection files
```

Comment:
> // resolve from redirection files [](start = 11, length = 34)

You can just put these logic to ResolveHref, it does not have any impact on ResolveContent

Reply:
If an markdown include a file that has been redirected, we should show a warning.

Adding the logic here will suppress the warning.

---
In reply to: [195022321](https://github.com/dotnet/docfx/pull/2886#discussion_r195022321) [](ancestors = 195022321)